test/integration/util/util.go

```diff
@@ -1157,21 +1157,3 @@ func NextPodOrDie(t *testing.T, testCtx *TestContext) *schedulerframework.Queued
 	}
 	return podInfo
 }
-
-// NextPod returns the next Pod in the scheduler queue, with a 5 seconds timeout.
```

Comment:
Why remove this? Unused?

Reply:
Yes, unused and it has the goroutine-leak problem as mentioned in the comment.

Reply:
Yes, please